README.md

```diff
@@ -22,7 +22,7 @@ $ composer install
 Install project :
 ```bash
 $ bin/console app:install
-$ yarn install && yarn run gulp
+$ yarn install && yarn run encore production (or yarn run encore dev-server --hot for developpement)
```

Comment:
@smildev Maybe we could add two yarn scripts on package.json:
```
"scripts": {
    "build": "yarn encore production"
    "dev": "yarn encore dev-server --hot"
     ....
  }
```

Reply:
development, not develo**pp**ment